.travis.yml

```diff
@@ -44,7 +44,17 @@ after_script:
     fi
 
 jobs:
+  allow_failures:
+    - php: nightly
```

Comment:
This is already configured:

https://github.com/doctrine/dbal/blob/54a6f7b3585e0cda5d5613200ea6bc6fadf04e95/.travis.yml#L230-L232